pkg/controller/transfer/pvc_test.go

```diff
@@ -309,6 +309,29 @@ var _ = Describe("PVC Transfer Tests", func() {
 			checkCompleteFalse(xfer, "Running", "")
 		})
 
+		It("Should create target even when a new source appears", func() {
```

Comment:
I tries to read the test but I do not understand what does it mean "a new source appears"

Reply:
hope above comment clarifies things